src/components/ProjectPreview.tsx

```diff
@@ -0,0 +1,18 @@
+import { PreviewText, PreviewWrapper, PreviewTitle } from './styles/ProjectPreview.styled'
+
+type Props = {
+  title: string
+  text: string
+  hasAnimation: boolean
+  fontSize: string
+  onClick: () => void
```

Comment:
might want to make this optional since the pages are not going to be up yet

Reply:
6df25e72fc18a34d467e2df2ee53f33329c995c6